app/views/admin/statistics_announcement_unpublishings/new.html.erb

```diff
@@ -1,34 +1,48 @@
-<% page_title "Unpublish statistics announcement" %>
+<% content_for :page_title, "Unpublish statistics announcement" %>
+<% content_for :context, @statistics_announcement.title %>
+<% content_for :title, "Unpublish statistics announcement" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @statistics_announcement)) %>
 
-<div class="stats-form-wrapper">
-  <h1>
-    <span class="stats-small-heading">
-      <%= @statistics_announcement.title %>
-    </span>
-    <span class="stats-heading">
-      Unpublish statistics announcement
-    </span>
-  </h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-body-lead">
+      <p>Take announcements off GOV.UK if they:</p>
+      <ul>
```

Comment:
can we use the component here https://components.publishing.service.gov.uk/component-guide/list#how-to-call-this-component

Reply:
Yeah, sorry missed this.